### PR TITLE
Uniform Rework

### PR DESCRIPTION
**Pull request by Realeasy90**
Uniform Rework

### CODE REVIEW

File: extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  class FDF_fat_m05_maasto_sin_2 {
+    class ItemInfo;
+  };
+
+  class FDF_fat_rs_m05_maasto_sin_2 {
+    class ItemInfo;
+  };
+
+  class CUP_U_B_USMC_FROG4_WMARPAT {
+    class ItemInfo;
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp]: Remove not needed base classes

File: extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    class ItemInfo: ItemInfo {
+      containerClass="Supply0";
+    };
+      // Change display name etc, just to find it better
+      displayName = "[PB] USMC (Woodland)";
+      scope=2;
+      author="CUP Team + [Pori Brig.] flaver";
+  };
+
+  class pvm_usmc_d: CUP_U_B_USMC_FROG4_DMARPAT {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp]: Remove usmc

File: extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+    class ItemInfo: ItemInfo {
+      containerClass="Supply0";
+    };
+      // Change display name etc, just to find it better
+      displayName = "[PB] USMC (Desert)";
+      scope=2;
+      author="CUP Team + [Pori Brig.] flaver";
+  };
+
+  class pvm_rm_d_r: CUP_U_B_BAF_DDPM_UBACSROLLEDKNEE {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp]: Remove baf

File: extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp [hunk 4]
```diff
@@ -0,0 +1,177 @@
+class CfgWeapons {
+  // Externals
+  class FDF_fat_m04_helle_sin_2 {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp]: You don't need FDF base classes here

File: extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+  class CUP_U_B_BAF_DDPM_UBACSLONGKNEE {
+    class ItemInfo;
+  };
+
+  class CUP_U_B_USMC_PilotOverall {
+    class ItemInfo;
+  };
+
+  class VSM_Multicam_Crye_SS_Camo {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp]: Don't need vsm here

File: extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    class ItemInfo: ItemInfo {
+      containerClass="Supply0";
+    };
+    // Change display name etc, just to find it better
+    displayName = "[PB] USMC (Pilot)";
+    scope=2;
+    author="CUP Team + [Pori Brig.] flaver";
+  };
+
+  class pvm_vms_multicam: VSM_Multicam_Crye_SS_Camo {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_fdf/CfgWeapons.hpp]: Remove vsm

File: extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+  class FDF_fat_m05_maasto_sin_2 {
+    class ItemInfo;
+  };
+
+  class FDF_fat_rs_m05_maasto_sin_2 {
+    class ItemInfo;
+  };
+
+  class CUP_U_B_USMC_FROG4_WMARPAT {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp]: Remove usmc

File: extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+    class ItemInfo: ItemInfo {
+      containerClass="Supply0";
+    };
+    // Change display name etc, just to find it better
+    displayName = "[PB] USMC (Pilot)";
+    scope=2;
+    author="CUP Team + [Pori Brig.] flaver";
+  };
+
+  class pvm_vms_multicam: VSM_Multicam_Crye_SS_Camo {
```
Comment (flaver12) [inline extendet veteran mod/addons/uniforms_uscm/CfgWeapons.hpp]: Remove vsm

File: extendet veteran mod/addons/uniforms_baf/config.cpp [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+            weapons[] = {};
+            requiredVersion = REQUIRED_VERSION;
+            requiredAddons[] = {};
+            author = "flaver";
+            VERSION_CONFIG;
+        };
+    };
+
+
+#include "CfgEventHandlers.hpp"
```
Comment (BaneKeldau) [inline extendet veteran mod/addons/uniforms_baf/config.cpp]: missin include "CfgWeapons.hpp"

File: extendet veteran mod/addons/uniforms_fdf/config.cpp [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+            weapons[] = {};
+            requiredVersion = REQUIRED_VERSION;
+            requiredAddons[] = {};
+            author = "flaver";
+            VERSION_CONFIG;
+        };
+    };
+
+
+#include "CfgEventHandlers.hpp"
```
Comment (BaneKeldau) [inline extendet veteran mod/addons/uniforms_fdf/config.cpp]: missing include "CfgWeapons.hpp"

File: extendet veteran mod/addons/uniforms_uscm/config.cpp [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+            weapons[] = {};
+            requiredVersion = REQUIRED_VERSION;
+            requiredAddons[] = {};
+            author = "flaver";
+            VERSION_CONFIG;
+        };
+    };
+
+
+#include "CfgEventHandlers.hpp"
```
Comment (BaneKeldau) [inline extendet veteran mod/addons/uniforms_uscm/config.cpp]: missing include "CfgWeapons.hpp"

File: extendet veteran mod/addons/uniforms_vsm/config.cpp [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+            weapons[] = {};
+            requiredVersion = REQUIRED_VERSION;
+            requiredAddons[] = {};
+            author = "flaver";
+            VERSION_CONFIG;
+        };
+    };
+
+
+#include "CfgEventHandlers.hpp"
```
Comment (BaneKeldau) [inline extendet veteran mod/addons/uniforms_vsm/config.cpp]: missing include "CfgWeapons.hpp"

**Pull request merged**